reduct/boolean-reduct/1-constraint-complement-subtraction.metta

```diff
@@ -0,0 +1,57 @@
+;; Name:                oneCcSub
+;; A function to remove an element from the point of application (POA) if it's negated form is 
+;;                      contained in the guard set of a commanding AND node (COM).
+;; Goal:                To remove a constraint that is not needed because it is commanded by 
+;;                      its negated form.
+;; POA:                 Any non-root AND node.
+;; Preconditions:       The POA’s guard set contains constraint X and the POA is commanded 
+;;                      by a terminal AND node COM whose guard set consists of the constraint 
+;;                      which is the opposite of X.
+;; Action:              Subtract the constraint X from the guard set of the POA.
+;;
+;; Example:             POA --> (AND B C), COM guard set ((NOT B) A) -- a negated literal of the POA is                        found in the COM guard set so the POA is commanded by this set and the common
+;;                      element is subtracted from the POA that member, B in this case. After the 
+;;                      transformation, the POA is now (AND C).
+(= (oneCcSub $comSet $poa $subExpr)
```

Comment:
It's not clear what the `$subExpr` is so can you add a comment or a corresponding example in addition to the POA in the above comment?

Reply:
May be you can rename it to `parent` if that's the case. But one last remark I have here is that the other transformation functions have parameter order of `parent`, `poa` and `comSet`. It's best to keep this same pattern for the sake of consistency.

Reply:
I have edited the file accordingly.

Reply:
Cool, can you reorder the parameters to match the order `parent`, `poa` and `com`?